### ISSUE TITLE
deprecated last_comment used

### ISSUE THREAD

**Issue by dlakatos847** [realityforge/knife-cookbook-doc#35]
My `Rakefile` section:

```
...
require 'knife_cookbook_doc/rake_task'
...
# With default options
KnifeCookbookDoc::RakeTask.new(:doc)
...
```

```
$> rake --tasks
[DEPRECATION] `last_comment` is deprecated.  Please use `last_description` instead.
rake doc                    # Generate cookbook documentation
...
$> rake doc
[DEPRECATION] `last_comment` is deprecated.  Please use `last_description` instead.
```

```
$> ruby --version
ruby 2.1.6p336 (2015-04-13 revision 50298) [x86_64-linux]
$> rake --version
rake, version 11.1.2
$> gem list | grep knife-cookbook-doc
knife-cookbook-doc (0.20.0)
```

**Comment by gitjoey** [comment]
👍  receiving this as well

**Comment by realityforge** [comment]
Fixed by 7d757e0621eec168e2faaf59f6fddab2fdbb09b5 - thanks for the report
